src/pages/Home/index.tsx

```diff
@@ -1,5 +1,16 @@
 import React, { ReactElement } from 'react';
+import styled from 'styled-components';
 
-const Home = (): ReactElement => <div>Home</div>;
+import { CreatePost } from '../../components/CreatePost/CreatePost';
+
+const Header = styled.div`
+  font-size: 1.5em;
+`;
+const Home = (): ReactElement => (
+  <div>
+    <Header>Home</Header>
+    <CreatePost />
```

Comment:
Non-blocking suggestion: Don't need to do it in this PR but the component composition and their responsibilities seems a bit off:
- `CreatePost` doesn't sound like a component and more like an operation - perhaps `KweetForm` or something along that line would be a better name
- It feels odd that the `CreatePost` or `KweetForm` component is rendering the list of Kweets - maybe it might be better

Reply:
I will create a separate issue to store data in context :)